### ISSUE TITLE
Unselected card style

### ISSUE THREAD

**Issue by Beto-Beto-Salamander** [iamvivekkaushik/DatePickerTimelineFlutter#59]
Hello, 
Can we change the unselected color for the card ?
I want to make it white, but it seems the default color is transparent

Thank you

**Comment by iamvivekkaushik** [comment]
You're right — as of v1.3.0 the unselected tile background is still hard-coded to transparent internally (`isSelected ? selectionColor : Colors.transparent`); only the selected tile gets `selectionColor`, and `deactivatedColor` only affects text. So there's no parameter for this yet — it's a fair request and I'm keeping it open as one, likely as part of a broader styling/per-date-builder pass (custom per-date widget builders aren't implemented yet either).

Closest workaround today is to paint behind the whole strip:
```
Container(
  color: Colors.white,
  child: DatePicker(
    DateTime.now(),
    selectionColor: Theme.of(context).primaryColor,
  ),
)
```
Caveat: that gives you a continuous white strip, not individually rounded white cards per date — true per-tile backgrounds need the API change tracked here.
